UnitTests/Schemas/IFC4X1/extruded-solid/src/extruded-solid.cpp

```diff
@@ -64,7 +64,12 @@ class ExtrudedSolidTest : public VisualTest {
 TEST_F(ExtrudedSolidTest, AllEntitiesAreRead) {
 	EXPECT_THAT(express_model->entities.size(), Eq(30));
 }
-
+//TODO:
+/*
+TEST_F(ExtrudedSolidTest, IFCHaveAnEssentialEntity) {
+	
+}
+*/
```

Comment:
Perfect, exactly what I had in mind :)

Reply:
Now it should also work after fa2b8df :)